spring-boot-admin-server-ui/src/main/java/de/codecentric/boot/admin/server/ui/config/AdminServerUiProperties.java

```diff
@@ -120,6 +120,18 @@ public class AdminServerUiProperties {
 
 	private PollTimer pollTimer = new PollTimer();
 
+	/**
+	 * Additional routes for home page redirecting filter. Any request to these routes
+	 * will be redirected to home page internally
+	 */
+	private List<String> additionalRoutes = new ArrayList<>();
```

Comment:
What should additional routes be used for?

We understand the need to add excludes, but why includes?

Reply:
Hm, that's a good question. I added them just like excludes and don't have an example. Maybe, it's better not add this property.

Reply:
ok please remove them

Reply:
Removed